test/web-platform-tests/index.js

```diff
@@ -176,6 +176,7 @@ describe("Web Platform Tests", () => {
     "html/editing/focus/sequential-focus-navigation-and-the-tabindex-attribute/focus-tabindex-default-value.html",
     // "html/infrastructure/urls/terminology-0/document-base-url.html", // we don't support srcdoc <base> correctly
     "html/obsolete/requirements-for-implementations/other-elements-attributes-and-apis/nothing.html",
+    "html/semantics/disabled-elements/disabledElement.html",
```

Comment:
This test already exists further down in the list.

Reply:
Ah, ordered lexically, not by folders. We should sort this whole list in general, the order really sucks currently.